src/lib.rs

```diff
@@ -118,10 +118,10 @@ impl<T> Webauthn<T> {
         let rp_id_hash = compute_sha256(config.get_relying_party_id().as_bytes());
         Webauthn {
             // Use a per-thread csprng
-            rng: rand::thread_rng(),
-            config: config,
-            pkcp: pkcp,
-            rp_id_hash: rp_id_hash,
+            rng: rand::rngs::StdRng::seed_from_u64(42),
```

Comment:
This is sadly, a security issue. By setting a static seed it means challenges are predictable. I have a better fix for this in my own branch that I'll push shortly, so you can probably consider at least reverting the changes here in lib.rs.